packages/xo-server/src/xo-mixins/subjects.js

```diff
@@ -231,6 +244,68 @@ export default class {
     })
   }
 
+  async registerUser2(providerId, { user: { id, name }, data }) {
```

Comment:
Shouldn't we make `registerUser` a small wrapper around this one?

Reply:
What do you mean by that exactly? Making `registerUser` check what the second parameter is before deciding what to do?

Reply:
I mean unifying the code path by making `registerUser` an interface for legacy code which calls `registerUser2` which implement a single logic :slightly_smiling_face: